apps/webapp/src/components/staking/account/index.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  VotingPowerAsIntegerPercentage,
+  bech32IdentityKey,
+  getIdentityKeyFromValidatorInfo,
+  getIdentityKeyFromValueView,
+  getValidatorInfoFromValueView,
+} from '@penumbra-zone/types';
+import { useStore } from '../../../state';
+import { stakingSelector } from '../../../state/staking';
+import { DelegationValueView } from './delegation-value-view';
+import { Card, CardContent, CardHeader, CardTitle } from '@penumbra-zone/ui';
+import { ValueView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1/asset_pb';
+import { ValueViewComponent } from '@penumbra-zone/ui/components/ui/tx/view/value';
+
+const getVotingPowerAsIntegerPercentage = (
+  votingPowerByValidatorInfo: Record<string, VotingPowerAsIntegerPercentage>,
+  delegation: ValueView,
+) =>
+  votingPowerByValidatorInfo[
+    bech32IdentityKey(getIdentityKeyFromValidatorInfo(getValidatorInfoFromValueView(delegation)))
+  ];
+
+export const Account = () => {
+  const { account, delegationsByAccount, unstakedTokensByAccount, votingPowerByValidatorInfo } =
+    useStore(stakingSelector);
+  const unstakedTokens = unstakedTokensByAccount.get(account);
+  const delegations = delegationsByAccount.get(account) ?? [];
+
+  return (
+    <div className='flex flex-col gap-4'>
+      {!!unstakedTokens && (
```

Comment:
Not strictly. But I've generally adopted the habit of using `!!` to coerce values to booleans in templates. Otherwise, you get the occasional unintended markup. For example, let's say you have a numeric value `balance`, and you write the following markup:

```tsx
<>
  {balance && <span>Balance: {balance}penumbra</span>}
</>
```

If `balance` is equal to `0`, then the following will be rendered:

```tsx
<>
  0
</>
```

...because the return value of a statement `somethingFalsey && somethingTruthy` is `somethingFalsey`. And since React will render any value to the page except for `null`, `false`, and `true` (IIRC), `somethingFalsey` (in this case, `0`) will get rendered to the page unintentionally. (See the "Pitfall" note [here](https://react.dev/learn/conditional-rendering#logical-and-operator-).)

Hope this makes sense, haha. I can definitely remove the `!!` if you want; I just feel like it's generally safer to include it.